corehq/messaging/smsbackends/turn/models.py

```diff
@@ -100,7 +101,11 @@ def _send_fallback_message(self, msg):
     def get_all_templates(self):
         config = self.config
         client = TurnBusinessManagementClient(config.business_id, config.business_auth_token)
-        return client.message_templates.get_message_templates()
+        try:
+            # For bad config values, this can throw an exception
+            return client.message_templates.get_message_templates()
+        except Exception:
```

Comment:
This will catch every exception, including ones that are a result of a bug in the code (misspelled variable name, etc). Is there a more targeted way of checking for the specific errors caused by bad config?

Reply:
I actually did try to catch the `JSONDecodeError` at first, but it didn't work. I changed it to the more general `Exception`, but having another look now I realised I imported `JSONDecodeError` from the wrong place. Works now!

Reply:
From your comments I again just realise it helps spending just that little extra time rounding off the code...